web/src/stores/sentences.ts

```diff
@@ -168,8 +168,15 @@ export namespace Sentences {
     }
   }
 
+  const localeSentences = ({ locale, sentences }: StateTree) => {
+    if (!sentences[locale]) {
+      return DEFAULT_LOCALE_STATE;
```

Comment:
This feels wrong, I think we should 'load' a default locale state into our state tree rather than do this.